.travis.yml

```diff
@@ -61,12 +61,16 @@ services:
 
 before_install:
   - sudo apt-get -qq update
-  - sudo apt-get install -y libseccomp-dev
+  - sudo apt-get install -y build-essential astyle golint gperf
+  - wget https://github.com/seccomp/libseccomp/releases/download/v2.5.0/libseccomp-2.5.0.tar.gz
+  - echo 463b688bf7d227325b5a465b6bdc3ec4 libseccomp-2.5.0.tar.gz | md5sum -c
+  - tar xf libseccomp-2.5.0.tar.gz
+  - cd libseccomp-2.5.0 && ./configure --prefix=/usr --libdir=/lib/x86_64-linux-gnu && make && sudo make install && cd ..
```

Comment:
No, it does not work for the "cgroup-systemd" job: that job is run on the ubuntu Travis host directly (and not within Docker or withing Vagrant like the other jobs in this file), so libseccomp is already installed and it has a previous version, so it fails.

`--libdir` is necessary on Ubuntu/Debian because the default libdir is not the same on Fedora and elsewhere, so it didn't override the old libseccomp before and it was failing. The `--libdir` is also necessary so that runc is built against the correct `.pc` file: `pkg-config` on Ubuntu/Debian is picking up the `pkgconfig` directory in a different place than `./configure` would put it.